{
   "assignee" : null,
   "body" : "In Bitcoin Core 0.11.1, this test script fails with `JSONRPC error: 66: insufficient priority` as shown below. The test passed in 0.11.0.\r\n\r\n```\r\nRunning 2nd level testscript pruning.py...\r\n  File \"/builddir/build/BUILD/bitcoin-0.11.1/qa/rpc-tests/test_framework/test_framework.py\", line 118, in main\r\n    self.run_test()\r\n  File \"/builddir/build/BUILD/bitcoin-0.11.1/qa/rpc-tests/pruning.py\", line 261, in run_test\r\n    self.create_big_chain()\r\n  File \"/builddir/build/BUILD/bitcoin-0.11.1/qa/rpc-tests/pruning.py\", line 76, in create_big_chain\r\n    self.mine_full_block(self.nodes[0], self.address[0])\r\n  File \"/builddir/build/BUILD/bitcoin-0.11.1/qa/rpc-tests/pruning.py\", line 253, in mine_full_block\r\n    txid = node.sendrawtransaction(signresult[\"hex\"], True)\r\n  File \"/builddir/build/BUILD/bitcoin-0.11.1/qa/rpc-tests/test_framework/authproxy.py\", line 126, in __call__\r\n    raise JSONRPCException(response['error'])\r\nInitializing test directory /tmp/testM1XDl2\r\nWarning! This test requires 4GB of disk space and takes over 30 mins (up to 2 hours)\r\nMining a big blockchain of 995 blocks\r\nJSONRPC error: 66: insufficient priority\r\nStopping nodes\r\nCleaning up\r\nFailed\r\n```",
   "closed_at" : "2015-10-20T10:53:58Z",
   "closed_by" : {
      "avatar_url" : "https://avatars.githubusercontent.com/u/126646?v=3",
      "events_url" : "https://api.github.com/users/laanwj/events{/privacy}",
      "followers_url" : "https://api.github.com/users/laanwj/followers",
      "following_url" : "https://api.github.com/users/laanwj/following{/other_user}",
      "gists_url" : "https://api.github.com/users/laanwj/gists{/gist_id}",
      "gravatar_id" : "",
      "html_url" : "https://github.com/laanwj",
      "id" : 126646,
      "login" : "laanwj",
      "organizations_url" : "https://api.github.com/users/laanwj/orgs",
      "received_events_url" : "https://api.github.com/users/laanwj/received_events",
      "repos_url" : "https://api.github.com/users/laanwj/repos",
      "site_admin" : false,
      "starred_url" : "https://api.github.com/users/laanwj/starred{/owner}{/repo}",
      "subscriptions_url" : "https://api.github.com/users/laanwj/subscriptions",
      "type" : "User",
      "url" : "https://api.github.com/users/laanwj"
   },
   "comments" : 2,
   "comments_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6834/comments",
   "created_at" : "2015-10-15T17:11:12Z",
   "events_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6834/events",
   "html_url" : "https://github.com/bitcoin/bitcoin/issues/6834",
   "id" : 111663991,
   "labels" : [],
   "labels_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6834/labels{/name}",
   "locked" : false,
   "milestone" : null,
   "number" : 6834,
   "repository_url" : "https://api.github.com/repos/bitcoin/bitcoin",
   "state" : "closed",
   "title" : "2nd level test pruning.py fails with \"insufficient priority\"",
   "updated_at" : "2015-10-20T10:53:58Z",
   "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6834",
   "user" : {
      "avatar_url" : "https://avatars.githubusercontent.com/u/198207?v=3",
      "events_url" : "https://api.github.com/users/error10/events{/privacy}",
      "followers_url" : "https://api.github.com/users/error10/followers",
      "following_url" : "https://api.github.com/users/error10/following{/other_user}",
      "gists_url" : "https://api.github.com/users/error10/gists{/gist_id}",
      "gravatar_id" : "",
      "html_url" : "https://github.com/error10",
      "id" : 198207,
      "login" : "error10",
      "organizations_url" : "https://api.github.com/users/error10/orgs",
      "received_events_url" : "https://api.github.com/users/error10/received_events",
      "repos_url" : "https://api.github.com/users/error10/repos",
      "site_admin" : false,
      "starred_url" : "https://api.github.com/users/error10/starred{/owner}{/repo}",
      "subscriptions_url" : "https://api.github.com/users/error10/subscriptions",
      "type" : "User",
      "url" : "https://api.github.com/users/error10"
   }
}
